libraries/AP_Scripting/lua_boxed_numerics.cpp

```diff
@@ -26,7 +26,7 @@ uint32_t coerce_to_uint32_t(lua_State *L, int arg) {
     { // float
         int success;
         const lua_Number v = lua_tonumberx(L, arg, &success);
-        if (success && v >= 0 && v <= UINT32_MAX) {
+        if (success && v >= 0 && v <= float(UINT32_MAX)) {
```

Comment:
we need to check this really works, especially for values close to UINT32_MAX. Can you please check that coercing UINT32_MAX doesn't produce an error?

Reply:
@WickedShell says he is happy with this